test/common/crypto/verify_signature_fuzz_test.cc

```diff
@@ -0,0 +1,43 @@
+#include "common/common/hex.h"
+#include "common/crypto/utility.h"
+#include "common/protobuf/utility.h"
```

Comment:
Is this needed? I think the proto validation is covered in `test_common/utility.h`

Reply:
Removed!